docker-compose.yml

```diff
@@ -74,3 +74,8 @@ services:
       default:
         aliases:
          - puppetdb.${DOMAIN:-internal}
+
+  puppetboard:
+    image: puppet/puppetboard
```

Comment:
I'm wondering if one of the other puppetboard containers would be better to use? We don't support that container and haven't published any updates to it in over a year, but there are a few on dockerhub that have been updated more recently 🤷‍♀

Reply:
@underscorgan 
I would suggest using images that Puppet controls. 
In that way not only **(1)** stability will be more easily guaranteed but also if **(2)** new features of the image are required in the Helm chart, they can be more easily directed and produced.

Reply:
@Xtigyro Puppet no longer supports the puppetboard image. We don't publish any updates to that container and don't maintain the puppetboard product.

Reply:
@underscorgan Then probably keep using the image from Puppet until `voxpopuli` releases their official image during the v0.2.x series as per those lines here: https://github.com/voxpupuli/puppetboard#docker-images

Reply:
@underscorgan 
I agree with @Xtigyro for the point he stated but more so that this PR is more about structure, I don't care so much about the puppetboard image in this specific PR, maybe we can make a very very near future compose/k8 related PR about swapping this image for a better one?  This was just to get something working, because for me without a puppetboard at all the entire compose/k8 structure is incomplete since we're flying blind-ish.  I also do agree that a more official image that is closer to the community is better.  Do we have plans to support something more officially?  All the versions I see on the hub are at 0.3.0 just as the unsupported puppet one is, but I'd need to do some more looking around.

Reply:
@underscorgan 

I've removed puppetboard for now to unblock this merge request. I'll continue using it locally for now, please re-review when possible.  I'll put puppetboard in a separate PR.